internalsdk/pro/proxy.go

```diff
@@ -1,115 +0,0 @@
-package pro
-
-import (
-	"bytes"
-	"compress/gzip"
-	"encoding/json"
-	"io"
-	"io/ioutil"
-	"net/http"
-	"net/http/httputil"
-	"strconv"
-	"strings"
-	"time"
-
-	"github.com/getlantern/lantern-client/internalsdk/common"
-	"github.com/getlantern/lantern-client/internalsdk/protos"
-)
-
-type proxyTransport struct {
```

Comment:
Not only was this unused, it's name was misleading.  It does not actually proxy requests.  I believe the misunderstanding may have been rooted in the use of `proxied.AsRoundTripper` by `NewHTTPClient` - see my comment about this [here](https://github.com/getlantern/lantern-client/pull/1191/files#r1780074359).